CMakeLists.txt

```diff
@@ -30,6 +30,30 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 include(common)
 include(environmentchecks)
 
+# Make sure that clcache/ccache gets used, if available.
+if(WIN32)
+    find_program(CLCACHE clcache)
+
+    if(CLCACHE)
+        set(CLCACHEWRAPPER ${PROJECT_BUILD_DIR}/clcachewrapper)
+
+        execute_process(COMMAND ${C_COMPILER} /O2 /Fe${CLCACHEWRAPPER} ${PROJECT_SOURCE_DIR}/cmake/clcachewrapper.c
```

Comment:
We don't enable to the C compiler so how does this work?

Reply:
FWIW, I was a bit slack (Friday drinks were looming) and I copied/pasted that code from OpenCOR, and didn't check it on Windows. In other words `C_COMPILER` is not defined here, but `CMAKE_C_COMPILER` is and can be used. See the revised code in commit 56ad7b1.